.gitlab/ci/.gitlab-ci.yml

```
@@ -42,7 +42,7 @@ variables:
   DEMISTO_CONNECTION_POOL_MAXSIZE: "180" # see this issue for more info https://github.com/demisto/etc/issues/36886
   SDK_REF: "master" # The default sdk branch to use
   OVERRIDE_ALL_PACKS: "false"
-  TEST_UPLOAD: "true"
+  TEST_UPLOAD: "false"
```

already fixed in the schedule
```suggestion
  TEST_UPLOAD: "true"
```